src/devtools/client/inspector/markup/new-markup.js

```diff
@@ -219,13 +241,13 @@ class MarkupView {
     // starting from the selected node itself since we want to display it expanded with
     // its children visible.
     while (currentNode) {
-      if (this.nodes.has(currentNode.objectId())) {
+      if (!this.nodes || this.nodes.has(currentNode.objectId())) {
         // We can stop importing once we encounter a node that is already known.
         break;
       }
 
       this.nodes.set(currentNode.objectId(), currentNode);
-      await this.addNodeToTree(currentNode, { isExpanded: true });
+      this.addNodeToTree(currentNode, { isExpanded: true });
```

Comment:
`addNodeToTree(...)` is synchronous, so no need to `await` it. It was previously marked as `async` but I removed that because it doesn't `await` anything.

Reply:
Nice